scripts/dev/pylint_checkers/qute_pylint/config.py

```diff
@@ -27,9 +27,11 @@
 import astroid
 from pylint import interfaces, checkers
 from pylint.checkers import utils
+from pathlib import Path
```

Comment:
Please...

- Move this up to the other Python stdlib imports
- Remove the unused `os` and `os.path` imports (not sure why `os` was imported in the first place
- Do `import pathlib` and use `pathlib.Path` instead (I prefer importing modules and sub-modules instead of importing things from modules)